Small improvements

PhMueller opened the pull request:
- Update the shutdown of a container
- Add script to delete hpobench related files
- Increase the supported python versions (@DaStoll)
-  Also, Singularity has decided to do random stuff. I had to update the download address for the singularity code.

PhMueller committed:
Fix the cleanup procedure.

Somehow, i received some ugly output and exceptions when a container was deleted.
This should do the trick. Hopefully, this is not something version (subprocess) related.

PhMueller committed:
Add script to clean hpobench related files

PhMueller committed:
Add support for >3.8

PhMueller committed:
Fix Github cant download release

PhMueller committed:
Install Singularity

PhMueller committed:
Fix singularity installation

KEggensperger commented on setup.py:
```diff
@@ -36,7 +36,7 @@ def read_file(file_name):
     packages=setuptools.find_packages(exclude=['*.tests', '*.tests.*',
                                                'tests.*', 'tests'],),
-    python_requires='>=3.6, <3.8',
+    python_requires='>=3.6, <=3.10',
```
Do we check test this also in the github actions?

KEggensperger commented on hpobench/util/clean_up_script.py:
```diff
@@ -0,0 +1,51 @@
+from hpobench import config_file
```
This scripts looks very useful, could you also link it in the Readme?

PhMueller committed:
Add ci testing for more python versions

Also test python versions 3.8 - 3.10

PhMueller committed:
Fix typos

PhMueller committed:
Add clean up script to Readme

PhMueller committed:
Fix dependencies.

Tensorflow==1.13 is not supported by >=python3.8, but we are not using cartpole in the tests anymore, so we can safely remove it.

PhMueller committed:
3.10 is currently not supported
